app/src/main/res/layout/activity_splash.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.design.widget.CoordinatorLayout
```

Comment:
You know, actually, I think it was auto-created when I used the wizard to create a new activity, but it's usually used to support some free out of the box material design animations and whatnot.